src/features/student/locales/fr.json

```
@@ -111,10 +111,41 @@
       }
     },
     "views": {
+      "studentActivityView": {
+        "breadcrumb": {
+          "current": {
+            "title": "AMS {activity}"
+          }
+        },
+        "title": "AMS - {activity}"
+      },
       "studentDeliverablesView": {
         "title": "(placeholder) Tous mes rendus"
       },
       "studentEducationActivitiesView": {
+        "breadcrumb": {
+          "current": {
+            "title": "Mes AMS"
+          }
+        },
+        "container": {
+          "studentDetailedActivityCard": {
+            "activityCount": "Acitvité {startedActivityCount}/{totalActivityCount} | Acitvités {startedActivityCount}/{totalActivityCount}",
+            "deliverables": {
+              "toSubmit": "1 rendu à soumettre pour le {date} | {count} rendus à soumettre pour le {date}",
+              "validated": "1 rendu validé | {count} rendus validés"
+            },
+            "skillCount": "0 compétence | 1 compétence | {count} compétences",
+            "status": {
+              "completed": "Terminée",
+              "inProgress": "En cours",
+              "notStarted": "Non initiée",
+              "submitted": "Soumise"
+            },
```

@p-beger Le status des activités ne doit pas être imbriqué ici, c'est des trads commun et metier, qui seront utilisé ailleurs https://www.figma.com/design/fA1dOIZs4Q3wTIJLSNQ5qk/AVENIRS-Maquettes?node-id=40002099-220589&m=dev

Tu peux les mettre dans la racine du fichier, dans un object `activities`
```suggestion
    "activities": {
      "status": {
        "completed": "Completed",
        "inProgress": "In progress",
        "notStarted": "Not started",
        "submitted": "Submitted"
      }
    }
```